scripts/local-setup.sh

```diff
@@ -8,16 +8,17 @@
 # - name of the kind k8s cluster (CLUSTER_NAME)
 # - Path to the Test Kustomization (KS_PATH)
 
-set -e
 # install flux
-if [ ! -f /usr/local/bin/flux ]; then
+which flux
+if [ $? -eq 1 ]; then
```

Comment:
kannst vielleicht mit
```suggestion
if which flux >/dev/null; then
```
vereinfachen